src/inatinqperf/benchmark/configuration.py

```diff
@@ -32,7 +32,7 @@ class VectorDatabaseParams(BaseModel):
     """Configuration for parameters initializing a Vector Database."""
 
     metric: Metric
-    index_type: NonEmptyStr
+    index_type: NonEmptyStr | None = None
```

Comment:
This should not change. If you want things to work, then specify the `index_type` in the config (we should make it mandatory to supply this). It also becomes self-documenting.